File: backend/LeitnerPlatform.Core/Entities/Banner.cs
using System;

namespace LeitnerPlatform.Core.Entities
{
    public class Banner
    {
        public Guid Id { get; set; }
        public string ImageUrl { get; set; } = string.Empty;
        public string? LinkUrl { get; set; }
        public int DisplayOrder { get; set; }
        public bool IsActive { get; set; } = true;
    }
}

File: mobile-app/lib/features/notifications/domain/entities/banner.dart
import 'package:equatable/equatable.dart';

class Banner extends Equatable {
  final String id;
  final String imageUrl;
  final String? linkUrl;
  final int displayOrder;
  final bool isActive;

  const Banner({
    required this.id,
    required this.imageUrl,
    this.linkUrl,
    required this.displayOrder,
    this.isActive = true,
  });

  @override
  List<Object?> get props => [id, imageUrl, linkUrl, displayOrder, isActive];
}
